project-reports/2023/2023-Q2-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Iroha
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Currently, active maintenance of Iroha 1 has stopped and there are no plans for major releases. However, discussions to release the current RC continue because it will require more than one team switching from everyday tasks to focus on this.
+
+All Iroha 2 developers are now fully onboarded and at maximum productivity.
+
+Since development of the Ursa project has been discontinued, the Iroha team is rewriting its essential parts to integrate in the Iroha codebase. In addition, thanks to Arun S M, the Iroha 2 BCE team will start collaborating with the Blockchain Explorer, a Hyperledger Lab, team.
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+Past quarter releases:
+
+* v2.0.0-pre-rc.13 (internal)
+* v2.0.0-pre-rc.14 (internal)
+* v2.0.0-pre-rc.15 (internal)
+
+# Overall Activity in the Past Quarter
+
+According to the [Iroha Activity Dashboard](https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Firoha/dashboard;subTab=technical?time=%7B%22from%22:%222023-01-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-03-31T07:00:00.000Z%22%7D) for the first quarter of 2023 (January-March), the Iroha repo has 129 PRs submitted by 28 contributors.
+
+## Iroha v1
+
+We are restoring the old documentation for the Iroha v1 project using a different hosting solution, as the previous solution was terminated. This is going to be used temporarily for support of the deprecated version of Iroha v1 documentation. 
+
+Iroha 1 has its own mentorship project: "[Extend queries with optional arguments](https://mentorship.lfx.linuxfoundation.org/project/06a25b14-a56e-4294-a89d-05f1dc74106c)".
+The mentors on this project are: Andrzej Gruntowski, Grzegorz Bazior and Aleksandr Petrosyan.
```

Comment:
Could you please add to each name a link to their github profile?

Reply:
I will, thanks for checking!